packages/relay/src/lib/constants.ts

```diff
@@ -38,4 +38,6 @@ export default {
 
     TYPE_CONTRACT: 'contract',
     TYPE_ACCOUNT: 'account',
+
+    MAX_FEE_HISTORY_RESULTS: 1024
```

Comment:
Let's make this number smaller for now. 
Say 10 since fees don't change often they are likely to be all the same.
Also given the rate of blocks produced i don't even think this number would capture the full cycle of fee updates even if it was done every hour.

Add an environment config for this so we can modify on the fly if needed